src/MongoDB/ReadConcern.c

```diff
@@ -101,6 +101,16 @@ PHP_METHOD(ReadConcern, getLevel)
 }
 /* }}} */
 
+/* {{{ proto array ReadConcern::bsonSerialize()
+   */
+PHP_METHOD(ReadConcern, bsonSerialize)
+{
+	const mongoc_read_concern_t *read_concern = phongo_read_concern_from_zval(getThis() TSRMLS_CC);
```

Comment:
I'm fine with not having warnings for unused function arguments though. Catching locally undefined vars is important though.